### PR TITLE
Production docker image

### PR DESCRIPTION
**Pull request by blabno**

### CODE REVIEW

File: src/main/java/io/bisq/api/app/ApiMain.java [hunk 1]
```diff
@@ -54,7 +54,6 @@ public static void main(String[] args) throws Exception {
 
         // We don't want to do the full argument parsing here as that might easily change in update versions
         // So we only handle the absolute minimum which is APP_NAME, APP_DATA_DIR_KEY and USER_DATA_DIR
-        BisqEnvironment.setDefaultAppName("bisq_api");
```
Comment (mrosseel) [inline src/main/java/io/bisq/api/app/ApiMain.java]: I suppose you removed it because there's another place where this is set?

Comment (blabno) [reply src/main/java/io/bisq/api/app/ApiMain.java]: The user can set it using program arguments. We should not overwrite that behavior.

Comment (mrosseel) [reply src/main/java/io/bisq/api/app/ApiMain.java]: agreed, but what if the user does not specify anything, it should have a default appname. EIther this default appname is specified somewhere else and we can delete this line, or it's not and then it should stay. If this name is assigned *after* the user's program argument is assigned that's also a problem of course.

**Comment by blabno** [comment]
Default is Bisq.

pon., 16 kwi 2018, 18:08 użytkownik mrosseel <notifications@github.com>
napisał:

**Comment by mrosseel** [comment]
do we want at startup to use the same dir as regular bisq or not, that's the question here I guess. 
Fine for me to use the bisq name, but there is a risk that people will mess up their bisq dir while testing the api.

**Comment by blabno** [comment]
I would use default so that we do not introduce new magic.

pon., 16 kwi 2018, 21:05 użytkownik mrosseel <notifications@github.com>
napisał:

**Comment by mrosseel** [comment]
ok, I'll merge now but this should be noted in the DOCS, that they are working on their 'real' Bisq dir if they don't supply a new appname

**Pull request merged**